src/components/Icon/Icon.tsx

```diff
@@ -56,7 +56,7 @@ export default function Icon({
     );
   } else if (source === 'placeholder') {
     contentMarkup = <div className={styles.Placeholder} />;
-  } else if (typeof source === 'string') {
+  } else {
```

Comment:
We don't need an if, elif, elif since we only have 3 type options (function, 'placeholder', string)